### PR TITLE
update docker-compose.yml with v0.2.0

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/area simulator
/kind documentation
/kind feature

#### What this PR does / why we need it:

Update docker-compose for -
https://github.com/kubernetes-sigs/kube-scheduler-simulator/releases/tag/simulator%2Fv0.2.0

Also update the release doc.

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

**Comment by sanposhiho** [comment]
/assign @utam0k 

Sorry to ping you everywhere 😅

**Comment by utam0k** [comment]
Don't worry, I'm online now.

File: RELEASE.md [hunk 1]
```diff
@@ -9,7 +9,9 @@
 7. Publish a draft release using the tag you created in 3.
 8. Perform the [image promotion process](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#image-promoter).
 9. Publish release.
-10. Email `kubernetes-sig-scheduling@googlegroups.com` to announce the release.
+10. Make sure the new version's image can be pulled and we can run the simulator correctly.
+11. Create a PR to update [docker-compose.yml](./docker-compose.yml) specifies the new release.
```
Comment (utam0k) [inline RELEASE.md:13]: Thanks for your update of the release flow 🙏 It's just a comment but ideally, we want to make it automatically by github actions in the future ;)

Comment (sanposhiho) [reply RELEASE.md:13]: Yup..

**Pull request merged**